tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -137,6 +137,29 @@ IF NOT "%CLRTestExitCode%"=="%CLRTestExpectedExitCode%" (
         ]]></Command>
         <Description>Run testcases under debugger.</Description>
       </BatchCLRTestExecutionScriptArgument>
+
+      <BatchCLRTestExecutionScriptArgument Include="env">
```

Comment:
This is already hooked up to the harness by the `__TestEnv` environment variable, which the harness sets directly. We could change this, but I think that it would require a rebuild of the tests each time a change to the environment script in use is desired. I think the current setup is good enough; the command line argument is only meant to make running a single test easier (same for `/coreroot`).